azure-pipelines.dfe.yml

```diff
@@ -1,7 +1,7 @@
 variables:
   BuildConfiguration: 'Release'
   CI: true
-  NODE_VERSION: 16.14.2
+  NODE_VERSION: 18.12.1
```

Comment:
Latest LTS Node is currently 18.17.0, should we bump this up again? Would need to be done across all relevant files of course

Reply:
Can do, wasn't sure if this would cause the networking issue previously mentioned but I'll give it a try now the LTS has been bumped